components/textfield/stories/textfield.stories.js

```diff
@@ -54,15 +58,7 @@ export default {
 		isKeyboardFocused,
 		size: size(["s", "m", "l", "xl"]),
 		isQuiet,
-		multiline: {
-			name: "Multiline",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "Component",
-			},
-			control: "boolean",
-		},
+		multiline: { table: { disable: true } },
```

Comment:
If we're creating a separate text area story, we can probably just delete this control, correct?

Reply:
It's hiding the controls